### PR TITLE
[community] Member profile : Deepak Dinesh 

### LINKED ISSUE #3836: [Community] Member Profile: Deepak Dinesh
**Current Behavior**

@deepakdinesh1123  has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @deepakdinesh1123  as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @deepakdinesh1123 
- Twitter: 
- LinkedIn: 
- Link to profile picture:

A detailed explanation on how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by sidharthviz**
**Description**

This PR fixes #3836 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 0198a2194c78324b9846d4cb10547580f608ad8d at: https://640028629f12fe30b16ec8b4--layer5.netlify.app

**Review by deepakdinesh1123 — APPROVED** [review]

**Comment by deepakdinesh1123** [comment]
@sidharthviz @gauravchauhan2403 @leecalcote Thank you very much for this means a lot.

**Comment by sidharthviz** [comment]
> @sidharthviz @gauravchauhan2403 @leecalcote Thank you very much for this means a lot.

Idk why it's not showing in build preview, Please correct me.

File: src/collections/members/deepak dinesh/index.mdx [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+name: Deepak Dinesh
+position: Contributor
+image_path: ./deepak.jpeg
+github: deepakdinesh1123
+twitter: https://twitter.com/LordDaemon13
+linkedin: https://www.linkedin.com/in/deepak-dinesh-6444b71a0/
+location: Karnataka, India
+bio: Backend Developer and open source contributor with an eye for technologies and frameworks that simplify the process of developing software.
+status: active
```
Comment (Chadha93) [inline src/collections/members/deepak dinesh/index.mdx:10]:
```suggestion
status: Active
```

**Comment by Chadha93** [comment]
> > @sidharthviz @gauravchauhan2403 @leecalcote Thank you very much for this means a lot.
> 
> Idk why it's not showing in build preview, Please correct me.

I fixed it, "active" is case sensitive.

**Comment by leecalcote** [comment]
Pretty awesome, @deepakdinesh1123 🎈

**Comment by l5io** [comment]
🚀 Preview for commit f571f6cdbdcd583ee7c35887754cf3195870f118 at: https://640109e0d8fa14284c0199c0--layer5.netlify.app

**Pull request merged**